doc/api/os.md

```diff
@@ -259,6 +259,8 @@ The properties available on the assigned network address object include:
 
 * `address` {string} The assigned IPv4 or IPv6 address
 * `netmask` {string} The IPv4 or IPv6 network mask
+* `broadcast` {string} The IPv4 or IPv6 broadcast address (not available
+  on all interface types)
```

Comment:
When we say "not available on all interface types"... does that mean it will return `undefined`?